test/powershell/Language/Parser/Parser.Tests.ps1

```diff
@@ -709,6 +709,32 @@ foo``u{2195}abc
                 @{ Script = "1dtb"; ExpectedValue = "1099511627776"; ExpectedType = [decimal] }
                 @{ Script = "1dpb"; ExpectedValue = "1125899906842624"; ExpectedType = [decimal] }
 
+            #SByte Integer notation
+                #Standard
+                @{ Script = "0y"; ExpectedValue = "0"; ExpectedType = [sbyte] }
+                @{ Script = "10y"; ExpectedValue = "10"; ExpectedType = [sbyte] }
+                @{ Script = "-10y"; ExpectedValue = "-10"; ExpectedType = [sbyte] }
+                @{ Script = "+10y"; ExpectedValue = "10"; ExpectedType = [sbyte] }
+                #Conversion from <Real>
+                @{ Script = "0.0y"; ExpectedValue = "0"; ExpectedType = [sbyte] }
+                @{ Script = "3.72y"; ExpectedValue = "4"; ExpectedType = [sbyte] }
+                @{ Script = "-3.72y"; ExpectedValue = "-4"; ExpectedType = [sbyte] }
+                #Exponential
+                @{ Script = "0e0y"; ExpectedValue = "0"; ExpectedType = [sbyte] }
+                @{ Script = "3e0y"; ExpectedValue = "3"; ExpectedType = [sbyte] }
+                @{ Script = "-3e0y"; ExpectedValue = "-3"; ExpectedType = [sbyte] }
+                @{ Script = "3e1y"; ExpectedValue = "30"; ExpectedType = [sbyte] }
+                @{ Script = "-3e1y"; ExpectedValue = "-30"; ExpectedType = [sbyte] }
+                #Hexadecimal
+                @{ Script = "0x0y"; ExpectedValue = "0"; ExpectedType = [sbyte] }
+                @{ Script = "0x41y"; ExpectedValue = "65"; ExpectedType = [sbyte] }
+                @{ Script = "-0x41y"; ExpectedValue = "-65"; ExpectedType = [sbyte] }
+                #Binary
+                @{ Script = "0b0y"; ExpectedValue = "0"; ExpectedType = [sbyte] }
+                @{ Script = "0b10y"; ExpectedValue = "2"; ExpectedType = [sbyte] }
+                @{ Script = "-0b10y"; ExpectedValue = "-2"; ExpectedType = [sbyte] }
+                @{ Script = "0b11111111y"; ExpectedValue = "-1"; ExpectedType = [sbyte] }
```

Comment:
Please remove the tests for binary suffix.

Reply:
Done! 😄